ipaclient/install/client.py

```diff
@@ -2055,9 +2056,9 @@ def install_check(options):
     global client_domain
     global cli_basedn
 
-    print("This program will set up FreeIPA client.")
-    print("Version {}".format(version.VERSION))
-    print("")
+    logcm("This program will set up FreeIPA client.")
+    logcm("Version %s", version.VERSION)
```

Comment:
Wrong conversion here, you need `%`, not `,`

Reply:
no, that's the correct form.